Library/Homebrew/dev-cmd/bump-formula-pr.rb

```diff
@@ -184,14 +189,13 @@ def bump_formula_pr
     elsif !new_url
       odie "#{formula}: no --url= argument specified!"
     else
-      resource_url = if requested_spec != :devel && new_url =~ /.*ftpmirror.gnu.*/
-        new_mirror = new_url.sub "ftpmirror.gnu.org", "ftp.gnu.org/gnu"
-        new_mirror
-      else
-        new_url
+      new_mirror = if requested_spec != :devel && new_url =~ %r{.*ftp.gnu.org/gnu.*}
```

Comment:
Could maybe make this a `case` statement if you anticipate more of these in future.